examples/dotnet/SeleniumDocs/Interactions/WindowsTest.cs

```diff
@@ -1,9 +1,49 @@
+using System;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
 
 namespace SeleniumDocs.Interactions
 {
     [TestClass]
-    public class WindowsTest : BaseTest
+    public class WindowsTest
     {
+        [TestMethod]
+        public void TestWindowCommands()
+        {
+            WebDriver driver = new ChromeDriver();
+            driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromMilliseconds(500);
+
+            // Navigate to Url
+            driver.Url="https://www.selenium.dev/selenium/web/window_switching_tests/page_with_frame.html";
+            //fetch handle of this
+            String currHandle = driver.CurrentWindowHandle;
+            Assert.IsNotNull(currHandle);
+
+            //click on link to open a new window
+            driver.FindElement(By.LinkText("Open new window")).Click();
+            //fetch handles of all windows, there will be two, [0]- default, [1] - new window
+            IList<string> windowHandles = new List<string>(driver.WindowHandles);
```

Comment:
HI @pallavigitwork ,

Thank you for the PR.

Looks like import `using System.Collections.Generic;` is missing for namespace IList<>, can you update pr with the same